intercoop_addons/account_move_import/wizard/wizard_account_move_import.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+from xlrd import open_workbook
+import base64
+from datetime import datetime
+
+from openerp import models, fields, api, _
+
+
+JOURNALS = [
+    ('ACMR', 'ACH'),
+    ('BNK1', 'CM'),
+    ('NDF', 'NDF'),
+    ('OD', 'OD'),
+    ('VTE1', 'VTE')
+]
+
+
+class AccountMoveImport(models.TransientModel):
+    _name = 'account.move.import'
+    _description = 'Import Account Moves import'
+
+    file = fields.Binary('File', required=True)
+    journal = fields.Selection(JOURNALS, 'Journal', required=True)
```

Comment:
Why not using a journal_id = fields.Many2one() field?

Reply:
For making the import easy because there are 39 journals and we have 5 fils to import with only 5 journals.